build.gradle.kts

```diff
@@ -43,7 +43,14 @@ allprojects {
                     }
                 }
                 "com.google.android.gms" -> useVersion(deps.versions.gms)
-                "org.jetbrains.kotlin" -> useVersion(deps.versions.kotlin)
+                "org.jetbrains.kotlin" -> {
+                    if (requested.name.startsWith("kotlin-stdlib-jre")) {
```

Comment:
what's all this?

Reply:
Starting from Kotlin 1.2, `kotlin-stdlib-jre` artifacts are deprecated in favor of `kotlin-stdlib-jdk` (more info here https://kotlinlang.org/docs/reference/whatsnew12.html#kotlin-standard-library-artifacts-and-split-packages). While we're using *-jdk artifacts internally, some external dependencies still can have obsolete dependencies, and Gradle keeps complaining of it. As soon as it's pure drop-in replacement, it's safe to do this resolution strategy.